Firestore/Swift/Tests/Integration/AggregationIntegrationTests.swift

```diff
@@ -18,9 +18,7 @@ import FirebaseFirestore
 import FirebaseFirestoreSwift
 import Foundation
 
-// TODO(sum/avg) remove `sumAvgIsPublic` from the directive below to enable these tests when sum/avg
-// is public
-#if sumAvgIsPublic && swift(>=5.5.2)
+#if swift(>=5.5.2)
```

Comment:
optional nit: swift 5.5.2 checks are no longer needed because we no longer support pre 5.5.2 versions